app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
```

Comment:
The variable `collection_of_coins` is incorrectly categorized as mutable. Sets in Python are mutable, so this is correct. However, ensure that this aligns with the task requirements.